httpsig_test.go

```diff
@@ -551,6 +623,69 @@ func Test_Signing_HTTP_Messages_AppendixC(t *testing.T) {
 	}
 }
 
+func Test_Signing_HTTP_Messages_AppendixC_Ed25519(t *testing.T) {
```

Comment:
I appreciate the trouble you went through to get a specific set of keys encoded/decoded. However, they originate from the blog post you linked to, and do not originate from the current draft of the spec ([rev 12](https://tools.ietf.org/html/draft-cavage-http-signatures-12#appendix-C)). I am worried these test names may give the incorrect impression that the key used in the test and the expectations in the test are both coming from the spec draft.

Perhaps simply renaming the two tests to be something along the lines of `TestSigningEd25519` and `TestVerifyingEd25519`? Then, if the spec authors include a ed25519 concrete example, it can be readily updated: PEM string, test expectations, and test name.

Thanks!

Reply:
Sure, np